tiatoolbox/annotation/storage.py

```diff
@@ -1371,17 +1456,24 @@ class SQLiteStore(AnnotationStore):
 
     Uses and rtree index for fast spatial queries.
 
+    Version History:
+    1.0.0:
+        Initial version.
+    1.0.1 (07/10/2022):
+        Added support for Area column and queries sorted/filtered by area.
```

Comment:
Just wanted to clarify how this is handling a store when loaded which does not have an area column> Is it just ignoring it and raising an error if you try to order on area? Or is it adding the area column at load time? If that latter, I am concerned that this would slow down loading of datasets if not using them for visualization.

Reply:
At the moment, the behaviour is that if you try to make a query that filters by area, and there isn't an area column, it raises an error stating that there is no area column.

Ordering by area happens by default if the area column is available. If it's not no ordering happens. At the moment there is no ability to specify what ordering happens. Probably in 2.0 we could add support for specifying what to sort on, be it area or perhaps some property.